react/components/ProductPrice/index.js

```diff
@@ -143,7 +144,7 @@ const ProductPrice = (props, context) => {
             handles.price_listPriceContainer
           )}
         >
-          {showLabels && (
+          {showLabels && !savingsAsPercentage && (
```

Comment:
Why is that necessary?

Reply:
Because `savingsAsPercentage` toggles between the default way of showing savings *OR* show savings as percentage. It should not render both ways.